src/api/question.ts

```diff
@@ -16,3 +16,16 @@ export interface Question {
     data: { [key: string]: string | string[] },
     required: boolean
 }
+
+type unittestError = {
```

Comment:
I *think* a common naming convention is to use PascalCase for types, and it'd be consistent with the below interface, but probably not a huge deal, or maybe there's some reasoning I don't understand.

Reply:
acb7df25